dev/com.ibm.ws.concurrent.persistent_fat/test-applications/schedtest/src/web/SchedulerFATServlet.java

```diff
@@ -376,6 +376,7 @@ public void testBlockAfterCancelByIdFE(PrintWriter out) throws Exception {
         final Phaser blocker = new Phaser(1);
         Future<Boolean> cancelAndBlockFuture = unmanagedExecutor.submit(new Callable<Boolean>() {
             public Boolean call() throws Exception {
+            	tran.setTransactionTimeout(12); //Increase transaction timeout to 12 seconds to accommodate slow test infrastructure. 
```

Comment:
This is a different transaction than the one for which the FFDC is reported. This one runs on an unmanaged executor. The other on a persistent executors InvokerTask, so the approach of tolerating the FFDC/error message on failover path can just be taken. The test otherwise runs fine and it ends up covering failover, for which it's reasonable to have messages.